crates/orderbook/openapi.yml

```diff
@@ -881,11 +881,16 @@ components:
             can be found in this object.
           allOf:
             - $ref: "#/components/schemas/OnchainOrderData"
-        executedSurplusFee:
-          description: Surplus fee that the limit order was executed with.
+        executedTotalFee:
```

Comment:
The `total` in the name seems a bit misleading to me.
I think most people would assume that this number would never increase once populated since it's already "total". But for partially fillable orders it would increase on every fill.
Do you think it would be less misleading if we just call it `executed_fee`?

Reply:
I thought of `executed_fee` but I am not too happy with none of them. `Executed_fee` seems to general, and one could ask himself `which one`. But if you prefer that one while I'm undecided, then it's probably worth to rename it.